gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/ServerGroupController.groovy

```diff
@@ -68,4 +69,11 @@ class ServerGroupController {
 
     return serverGroupDetails
   }
+
+  @ApiOperation(value = "Retrieve the details of a list of server groups", hidden = true)
+  @RequestMapping(value = "/getServerGroupsDetailsForList", method = RequestMethod.POST)
```

Comment:
Per the comment in https://github.com/spinnaker/clouddriver/pull/1952, I don't love the name or that this is a POST, but don't want to have to batch the calls given the length of the strings involved and the potential number of calls needing to be made.